apps/ui/components/single-search/FilterTagList.tsx

```diff
@@ -0,0 +1,135 @@
+    "--tag-background": "transparent",
+  },
+})``;
+
+const FilterTagList = ({
+  formValueKeys,
+  formValues,
+  removeValue,
+  getFormSubValueLabel,
+}: FilterTagProps) => {
+  const filterOrder = [
+    "textSearch",
+    "begin",
+    "end",
+    "after",
+    "before",
+    "duration",
+    "minPersons",
+    "maxPersons",
+    "reservationUnitType",
+    "unit",
+    "purposes",
+    "equipments",
+  ];
+  const multiSelectFilters = [
+    "unit",
+    "reservationUnitType",
+    "purposes",
+    "equipments",
+  ];
+  const { t } = useTranslation();
+  const durationTranslation = (duration: number): string => {
+    let unit = t("common:abbreviations.hour", { count: inHours(duration) });
+    if (duration <= 90) {
+      unit = t("common:abbreviations.minute", { count: duration });
+    }
+    return t("searchForm:filters.duration", { unit });
+  };
+  return (
+    <TagControls>
+      <FilterTags data-test-id="search-form__filter--tags">
+        {formValueKeys
+          .sort((a, b) => filterOrder.indexOf(a) - filterOrder.indexOf(b))
+          .map((formValueKey) => {
+            if (formValueKey === "showOnlyAvailable") return null;
+            const label = t(`searchForm:filters.${formValueKey}`, {
+              label: formValueKey,
+              value: formValues[formValueKey],
+              count: Number(formValues[formValueKey]),
+            });
+            return multiSelectFilters.includes(formValueKey) ? (
+              (formValues[formValueKey] ?? "").split(",").map((subValue) => (
+                <StyledTag
+                  id={`filter-tag__${formValueKey}-${subValue}`}
+                  onClick={() =>
+                    removeValue && removeValue([subValue], formValueKey)
+                  }
+                  onDelete={() =>
+                    removeValue && removeValue([subValue], formValueKey)
+                  }
+                  key={`${formValueKey}-${subValue}`}
+                  deleteButtonAriaLabel={t(`searchForm:removeFilter`, {
+                    value: getFormSubValueLabel(formValueKey, subValue),
+                  })}
+                >
+                  {getFormSubValueLabel(formValueKey, subValue)}
+                </StyledTag>
+              ))
+            ) : (
+              <StyledTag
+                id={`filter-tag__${formValueKey}`}
+                onDelete={() => removeValue && removeValue([formValueKey])}
+                key={formValueKey}
+                deleteButtonAriaLabel={t(`searchForm:removeFilter`, {
+                  value: label,
+                })}
+              >
+                {formValueKey === "duration" &&
+                !Number.isNaN(Number(formValues.duration))
+                  ? durationTranslation(Number(formValues.duration))
+                  : label}
+              </StyledTag>
+            );
+          })}
+      </FilterTags>
+      {formValueKeys.length > 0 && (
+        <ResetButton
+          aria-label={t("searchForm:resetForm")}
```

Comment:
aria-label is unnecessary if the inner html is same as the label

Reply:
Okiedokie - will remove it. This is how it was done before, I simply refactored it from `SearchForm` to it's own component for clarity, as `SearchForm` started to be really stuffed and hard to read.